praw/models/reddit/modmail.py

```diff
@@ -182,17 +182,33 @@ def highlight(self):
         """
         self._reddit.post(API_PATH["modmail_highlight"].format(id=self.id))
 
-    def mute(self):
+    def mute(self, duration=None):
         """Mute the non-mod user associated with the conversation.
 
+        :param duration: Duration of mute in days. Valid options are 3, 7, or 28.
```

Comment:
Rather than `duration` it might make sense for this parameter to be called `num_days`.

Reply:
Good point.